src/CustomProviders/examples/Get-AzCustomProvider.md

```diff
@@ -0,0 +1,30 @@
+### Example 1: List all Custom Providers in a subscription
+```powershell
+PS C:\> Get-AzCustomProvider
+
+Location  Name             Type
+--------  ----             ----
+West US 2 Namespace.Type   Microsoft.CustomProviders/resourceproviders
+East US 2 Namespace2.Type  Microsoft.CustomProviders/resourceproviders
+```
+
+Lists all the custom providers in a subscription
+
+### Example 2: Get a single custom provider
+```powershell
+PS C:\> Get-AzCustomProvider -ResourceGroupName myRg -Name Namespace.Type | Format-List
+
+Action            :
+Id                : /subscriptions/xxxxx-yyyyy-xxxx-yyyy/resourceGroups/mc-cp01/providers/Microsoft.CustomProviders/resourceproviders/Namespace.Type
+Location          : West US 2
+Name              : Namespace.Type
+ProvisioningState : Succeeded
+ResourceType      : {CustomRoute1, associations}
+Tag               : Microsoft.Azure.PowerShell.Cmdlets.CustomProviders.Models.Api20180901Preview.ResourceTags
```

Comment:
Should we have the tag list here?

Reply:
Agreed. It's a design flaw and I believe there's currently no workaround. 
I've logged an issue here: https://github.com/Azure/autorest.powershell/issues/618